format/applebookmark/apple_bookmark.jq

```diff
@@ -0,0 +1,34 @@
+def _apple_bookmark_torepr:
+  def _f:
+    ( if .type == "String" then .data | tovalue
+      elif .type == "Data" then .data | tovalue
+      elif .type == "Byte" then .data | tovalue
+      elif .type == "Short" then .data | tovalue
+      elif .type == "Int" then .data | tovalue
+      elif .type == "Long" then .data | tovalue
+      elif .type == "Float" then .data | tovalue
+      elif .type == "Double" then .data | tovalue
+      elif .type == "Date" then .data | tovalue
+      elif .type == "BooleanFalse" then false
+      elif .type == "BooleanTrue" then true
+      elif .type == "Array" then 
+        ( .data
+        | map(.record | _f)
+        )
+      elif .type == "Dictionary" then
+        ( .data
+        | map({key: (.key | _f), value: (.value | _f)})
+        | from_entries
+        )
+      elif .type == "UUID" then .data | tovalue
+      elif .type == "URL" then .data | tovalue
+      elif .type == "RelativeURL" then
+		.data | map(.record.data)
```

Comment:
Some tab character sneak in here i think. 2 space per indent seems to work nice with jq's syntax